src/init.cpp

```diff
@@ -1216,7 +1218,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     assert(!node.fee_estimator);
     // Don't initialize fee estimation with old data if we don't relay transactions,
     // as they would never get updated.
-    if (!ignores_incoming_txs) {
+    if (!peerman_opts.ignore_incoming_txs) {
```

Comment:
No strong opinion, but it would be perfectly fine to just do
```suggestion
    if (!args.GetBoolArg("-blocksonly", DEFAULT_BLOCKSONLY)) {
```
here and then keep the peerman opts were they are right now.

Reply:
Yeah, that would be my approach if not the current one. But going to keep as is for now since everyone seems to be happy with the current approach.